WordPressKit/AtomicAuthenticationServiceRemote.swift

```diff
@@ -0,0 +1,92 @@
+import Foundation
+
+struct RemoteCookie {
```

Comment:
Is this here for future tasks?

If not, can we delete it?

Reply:
Good catch, that was left in by mistake.